main.py

```diff
@@ -37,7 +37,9 @@
 bot = commands.Bot(command_prefix='.')
 FILENAME = None
 
-commandFolder="commands_to_load."
+with open('commands_to_load/cogs.json') as c:
```

Comment:
you relocated where the code was but apparently forgot to migrate the logging `logger.info('[main.py] loading cog names from json file')` with it?

Reply:
ok